everest

```diff
@@ -394,13 +385,13 @@ MSG
         red "Warning: stale /cygdrive/c/ocamlmgw64; continue? [Yn]"
         prompt_yes true "exit 1"
       fi
-      (cd c:/ && wget -O - $OCAML_TARBALL | tar xjvf -)
-      (cd ~ && wget -O - $OPAM_TARBALL | tar xjvf -)
-      (cd ~/.opam/system/lib && find . -iname '*.cmxs' -exec chmod a+x {} \;)
-      (cd ~/.opam/system/lib &&
-        echo -e "destdir=\"$(cygpath -m $(pwd))\"\npath=\"$(cygpath -m $(pwd))\"" > findlib.conf.new &&
-        tail -n +3 findlib.conf >> findlib.conf.new &&
-        mv findlib.conf.new findlib.conf)
+      # Download and Install OPAM
+      (wget $OPAM_URL)
+      (tar -xf 'opam64.tar.xz')
+      (rm -f 'opam64.tar.xz')
+      (bash opam64/install.sh)
+      (opam init mingw 'https://github.com/fdopen/opam-repository-mingw.git' --comp $OPAM_VERSION --switch $OPAM_VERSION)
+      eval $(opam config env)
```

Comment:
does this install `opam` and `flexlink` in `/usr/local/bin` or do we need to add `$(pwd)/opam64` to the `PATH`?

Reply:
it does install on /usr/local/bin